docs/products/mysql/concepts/high-availability-mysql.md

```diff
@@ -0,0 +1,131 @@
+---
+title: High availability of Aiven for MySQL®
+sidebar_label: High availability
+---
+
+Aiven for MySQL® is available on a variety of plans, offering different levels of high availability. The selected plan defines the features available, and a summary is provided in the table below:
+
+| Plan         | High availability features                                      | Backup history | Uptime  |
+| ------------ | --------------------------------------------------------------- | -------------- | ------- |
+| **Hobbyist** | Single node with limited availability                           | 2 days         | 98.5 %  |
+| **Startup**  | Single node with limited availability                           | 2 days         | 99 %    |
+| **Business** | Two nodes (primary + standby: regular high availability)        | 14 days        | 99.9 %  |
+| **Premium**  | Three nodes (primary + 2 x standby: strong high availability)   | 30-day         | 99.99 % |
+
+## About primary and standby nodes
+
+Aiven's Business and Premium plans offer a primary node and standby nodes. A standby
+service is useful for multiple reasons:
+
+-   Provides another physical copy of the data in case of hardware,
+    software, or network failures
+-   Reduces the data loss window in failure scenarios
+-   Provides higher availability as failure recovery leverages an already provisioned
+    standby node and does not suffer the delay of restoring a backups
+-   Can be used for low-availability or occasionnal read-only queries
+    server
+
+## Failure handling
+
+### Minor failures
+
+Minor failures, such as service process crashes or temporary loss of
+network access, are handled automatically by Aiven in all plans without
+any major changes to the service deployment. The service automatically
+resumes normal operation once the crashed process is automatically restarted or when
+network access is restored.
+
+### Severe failures
+
+Severe failures occur, for example, when a node is lost entirely in case of hardware
+or severe software problems, such as an unrecoverable crash-looping condition. Severe
+failures require radical recovery measures. The Aiven monitoring infrastructure
+automatically detects a failing node both when the node starts reporting issues in the
+self-diagnostics or when stops communicating. The monitoring infrastructure automatically
+schedules the creation of a replacement node unless there is a standby to which a failover
+can be performed.
+
+:::note
+In case of database failover, the **Service URI** of your service
+remains the same; only the IP address changes to point to the new
+primary node.
+:::
+
+### RTO and RPO
+
+- With standby, recovery time objective (RTO) is quick, and recovery point objective (RPO)
+  is close to zero loss.
+- Without standby, RPO loses 5 minutes of data, and RTO is variable as it involves
+  restoring a backup and applying binlogs, which depends on the database size and the
+  write load (up to 6 hours or more in edge cases).
```

Comment:
Not sure why we put *6* hours here. I'd say something like "up to several hours".